docker/deploy/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM ubuntu:xenial
```

Comment:
I figured I would go with the latest release that has long-term support, which is a sign of stability. Another reasonable choice would be 14.04, but if we're part of a forward-looking ecosystem we may find that using the newer release causes fewer problems.